content/docs/wrp/simple-messages.md

```diff
@@ -96,9 +104,10 @@ great at sending metrics, publishing a report, sending an SOS.
 Name | Description
 -----|--------------
 msg_type | The message type for the simple event.  (**SHALL** be 4.)
-source | The device_id name of the device originating the request or response.
-dest | The device_id name of the target device of the request or response.
-content_type | (optional) The media type of the payload.
+source | The device_id name of the device originating the event (i.e. mac:112233445566/config)
```

Comment:
```suggestion
source | The locator information of the creator of the event (i.e. mac:112233445566/config)
```
As I'm sure you can tell by now, I'm not a huge fan of using the term `device_id`. 🙂